src/oic/oic/provider.py

```diff
@@ -1567,7 +1568,7 @@ def create_registration(self, authn=None, request=None, **kwargs):
         )
         try:
             request = request_cls().deserialize(request, "json")
-        except ValueError:
+        except DecodeError:
```

Comment:
from_dict() may raise ValueError if values in the dict are unexpected (e.g. bool values that are not bool). Those errors would now raise here. So maybe catch both.